modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -0,0 +1,60 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distereoibution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#ifndef OPENCV_GAPI_STEREO_HPP
+#define OPENCV_GAPI_STEREO_HPP
+
+#include <opencv2/gapi/gmat.hpp>
+#include <opencv2/gapi/gscalar.hpp>
+#include <opencv2/gapi/gkernel.hpp>
+
+namespace cv { 
+namespace gapi {
+namespace stereo {
+
+enum OutputFormat {
+    disparity,
```

Comment:
Capital letters are usually used + `enum class`

Reply:
Done

Reply:
Capital letters are usually used for enum values and mixed-case style for enum name, @TolyaTalamanov , do you agree?

Reply:
`OUTPUT_FORMAT` -> `OutputFormat`
`disparity` -> `DISPARITY`